Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx

```diff
@@ -423,7 +423,7 @@ const EquipmentContainer = (props: IProps): JSX.Element => {
       );
       toast.success('Equipment updated successfully!');
     } catch (e) {
-      toast.error('Error editing equipment.');
```

Comment:
If you want to do it quick and easy, just do `toast.error('Error editing equipment. ' + e);`
But @srkirkland may have a better way.

Reply:
> If you want to do it quick and easy, just do `toast.error('Error editing equipment. ' + e);`
> But @srkirkland may have a better way.

Actually, I don't think that works if the BadRequest doesn't have text. I think it does this if you do that change:
![image](https://user-images.githubusercontent.com/469010/109875463-4f3d8c80-7c25-11eb-95f2-870045992116.png)

Reply:
yea you'd want to use `e.Message` instead of just `e` (since e includes stack trace and other stuff).